backend/app/tests/services/collections/test_create_collection.py

```
@@ -188,3 +189,67 @@ def test_execute_job_success_flow_updates_job_and_creates_collection(
+    )
+
+    req = CreationRequest(
+        model="gpt-4o",
+        instructions="string",
+        temperature=0.0,
+        documents=[],
+        batch_size=1,
+        callback_url=None,
+    )
+
+    mock_client = mock_get_openai_client.return_value
```

Ruff (0.14.0)

Local variable `mock_client` is assigned to but never used

Remove assignment to unused variable `mock_client`

(F841)